### PR TITLE
Connected Object details page to navigation

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

Connected Object details page to navigation schema. Also connected file details view with backend

**NOTE:** There is a known issue on clicking back button / hot reload where information is not loaded on screen. Will adress this in the next PR

### CODE REVIEW

File: pkg/acl/endpoints_test.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func TestGetAuthorizedEndpoints(t *testing.T) {
 					"s3:*",
 				},
 			},
-			want: 18,
+			want: 20,
```
Comment (dvaldivia) [inline pkg/acl/endpoints_test.go:94]:
```suggestion
			want: 18,
```

File: pkg/acl/endpoints_test.go [hunk 2]
```diff
@@ -72,7 +72,7 @@ func TestGetAuthorizedEndpoints(t *testing.T) {
 					"admin:*",
 				},
 			},
-			want: 15,
+			want: 17,
```
Comment (dvaldivia) [inline pkg/acl/endpoints_test.go:75]:
```suggestion
			want: 15,
```

**Review by dvaldivia — COMMENTED** [review]
This will fix testrs

**Review by dvaldivia — APPROVED** [review]

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx [hunk 3]
```diff
@@ -311,6 +311,11 @@ const ListObjects = ({
   const openPath = (idElement: string) => {
     const currentPath = get(match, "url", "/object-browser");
 
+    // const tableActions = [
+    //   { type: "view", to: `/object-browser/${selectedBucket}`, sendOnlyId: true },
+    //   { type: "download", onClick: downloadObject, sendOnlyId: true },
+    //   { type: "delete", onClick: confirmDeleteObject, sendOnlyId: true },
+    // ];
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx:318]: please delete this commented code

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ObjectRouting.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  routesList: Route[];
+  match: any;
+}
+
+const ObjectRouting = ({ routesList, match }: ObjectRoutingProps) => {
+  const currentItem = routesList[routesList.length - 1];
+
+  useEffect(() => {
+    console.log(match);
+  }, [match]);
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ObjectRouting.tsx:24]: is this need it?

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ObjectRouting.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+  useEffect(() => {
+    const url = get(match, "url", "/object-browser");
+
+    console.log("URL", url, "ROUTE", routesList[routesList.length - 1].route);
+
+    if (url !== routesList[routesList.length - 1].route) {
+      console.log("asdf");
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ObjectRouting.tsx:32]: please delete this

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/SetRetention.tsx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+  const dateElement = useRef<IRefObject>(null);
+
+  const dateFieldDisabled = () => {
+    return !(statusEnabled && (type === "governance" || type === "compliance"));
+  };
+
+  const onSubmit = (e: React.FormEvent) => {
+    e.preventDefault();
+    console.log("on submit");
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ObjectDetails/SetRetention.tsx:52]: please delete this

File: portal-ui/src/screens/Console/Common/FormComponents/DateSelector/DateSelector.tsx [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+              <MenuItem value={"1"}>1</MenuItem>
+              <MenuItem value={"2"}>2</MenuItem>
+              {/* {options.map((option) => (
+              <MenuItem
+                value={option.value}
+                key={`select-${name}-${option.label}`}
+              >
+                {option.label}
+              </MenuItem>
+            ))} */}
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Common/FormComponents/DateSelector/DateSelector.tsx:209]: please delete this

File: portal-ui/src/screens/Console/Common/FormComponents/DateSelector/DateSelector.tsx [hunk 8]
```diff
@@ -0,0 +1,245 @@
...
+              <MenuItem value={"2020"}>2020</MenuItem>
+              <MenuItem value={"2021"}>2021</MenuItem>
+              {/* {options.map((option) => (
+              <MenuItem
+                value={option.value}
+                key={`select-${name}-${option.label}`}
+              >
+                {option.label}
+              </MenuItem>
+            ))} */}
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/Common/FormComponents/DateSelector/DateSelector.tsx:236]: please delete this

**Review by Alevsk — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Common/FormComponents/DateSelector/DateSelector.tsx (hunk 7 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Common/FormComponents/DateSelector/DateSelector.tsx:209]: This is going to be needed for next step of date selector

File: portal-ui/src/screens/Console/Common/FormComponents/DateSelector/DateSelector.tsx (hunk 8 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Common/FormComponents/DateSelector/DateSelector.tsx:236]: This is going to be needed for next step of date selector

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ObjectRouting.tsx (hunk 4 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ObjectRouting.tsx]: No, we don't need it anymore

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx (hunk 3 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ListObjects.tsx]: done

File: portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ObjectRouting.tsx (hunk 5 above)
Comment (bexsoft) [reply portal-ui/src/screens/Console/Buckets/ListBuckets/Objects/ListObjects/ObjectRouting.tsx]: done

**Review by Alevsk — APPROVED** [review]

**Review by kaankabalak — APPROVED** [review]

**Comment by cesnietor** [comment]
tested LGTM

**Pull request merged**